src/data-workspace/table/table.js

```diff
@@ -0,0 +1,88 @@
+import {
+    DataTable,
+    TableHead,
+    DataTableRow,
+    DataTableColumnHeader,
+    TableBody,
+    DataTableCell,
+} from '@dhis2/ui'
+import PropTypes from 'prop-types'
+import React from 'react'
+import styles from './table.module.css'
+
+// Needs to have the same width as the table, so can't use the one from
+// @dhis2/ui
+const DataTableToolbar = ({ children, columns }) => (
+    <tr>
+        <th className={styles.titleCell} colSpan={columns.toString()}>
+            {children}
+        </th>
+    </tr>
+)
+
+DataTableToolbar.propTypes = {
+    children: PropTypes.any.isRequired,
+    columns: PropTypes.number.isRequired,
+}
+
+const Table = ({ title, columns, rows }) => (
+    <>
+        <DataTable className={styles.dataTable} width="auto">
+            <TableHead>
+                <DataTableToolbar columns={columns.length}>
+                    {title}
+                </DataTableToolbar>
+                <DataTableRow>
```

Comment:
The 'data element' column should be fixed (see design notes https://docs.google.com/document/d/1zHa9A8bTZyfCfIJpgRGsghnDl2G7AerypXPwtGemsrA/edit?disco=AAAAgyYJpJg)